lib/tarantool_server.py

```diff
@@ -449,7 +449,7 @@ def seek_wait(self, msg, proc=None, name=None):
         while True:
             if os.path.exists(self.path):
                 break
-            time.sleep(0.001)
+            gevent.sleep(0.001)
```

Comment:
Regarding commit message:

> Use gevent sleep

The title should give an idea about the change for someone, who didn't read the patch, in some concise way. The most descriptive way is often to describe what was changed from a user point of view.

> Was changed `time.sleep` to `gevent.sleep` to allow current greenlet to
> sleep and others to run.

This information is more or less obvious from the patch (however, sure, good to mention). However the commit message lacks non-obvious details: what is the greenlet we should unblock, what is the problem (and what are circumstances when it appears), what will be changed for a user?

Also, please, mark it as part of #276.

A commit message is written once, but will be read many times. It is worthful to maximize its usefulness. Especially at the point, when you remember every detail about the problem, about the patch, used tricks and so on.

I spin a lot around different code and I'm under a strict impression that development is mostly about expressing ideas rather than about tweaking code to make it work. At least in long living projects.

Reply:
I hope I fixed it well.

Reply:
The title does not explain changes from a user perspective, but it is more or less descriptive anyway. Okay.

Please, mark the first commit as `Part of #276`, not `Fixes #276`. Just from the common sense: we cannot fix an issue several times in row.

Generally speaking a patch series should be marked this way (or something equivalent):

* Part of #xxx.
* Part of #xxx.
* Fixes #xxx.
* Follows up #xxx.
* Follows up #xxx.

I would mark the last commit as `Follows up #276`, because we come into the need of negative testing during work on the issue.

Reply:
Got it